rules/vue.js

```diff
@@ -0,0 +1,45 @@
+module.exports = {
+  root: true,
+  env: {
+    browser: true,
+    node: true
+  },
+  parser: 'vue-eslint-parser',
+  plugins: ['import'],
+  parserOptions: {
+    ecmaVersion: 2022,
+    sourceType: 'module',
+    ecmaFeatures: {
+      jsx: true,
+      modules: true
+    }
+  },
+  globals: {
+    defineProps: 'readonly',
+    defineEmits: 'readonly',
+    defineExpose: 'readonly',
+    globalThis: 'readonly'
+  },
+  settings: {
+    'import/resolver': {
+      alias: {
+        map: [['@', './src']],
+        extensions: ['.js', '.vue', '.yml', '.scss', '.ts']
+      },
+      node: {
+        extensions: ['.js', '.jsx', '.ts', '.tsx']
+      }
+    }
+  },
+  ignorePatterns: ['package.json', 'dist/'],
+  extends: [
+    'airbnb',
+    'eslint:recommended',
+    'plugin:vue/vue3-recommended',
+    'prettier',
+    'plugin:import/errors'
+  ],
+  rules: {
+    'vue/no-v-html': 0
```

Comment:
This is missing a chunk of the rules we need for the memberhub. I however don't know if we use the default values and that's why we don't need them here

https://github.com/TIGNUM/qot-web-application/blob/develop/.eslintrc.js

![image](https://user-images.githubusercontent.com/1942073/163151955-48713099-ad3d-41a7-9478-2aa0a3298ab0.png)

Reply:
This is a WIP, I'm working in it ^^